cli_reference/admin_cli_operations.adoc

```diff
@@ -0,0 +1,171 @@
+= Administrator CLI Operations
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+This topic provides information on the administrator CLI operations and their syntax. You must link:get_started_cli.html[setup and login] with the CLI before you can perform these operations.
+
+[[common-operations]]
+
+== Common Operations
+The CLI allows interaction with the various objects that are managed by
+OpenShift. Many common `oadm` operations are invoked using the following syntax:
+
+----
+$ oadm <action> <option>
+----
+
+This specifies:
+
+- An `_<action>_` to perform, such as `new-project` or `router`.
+- An available `_<option>_` to perform the action on as well as a value for the option. Options include `--output` or `--credentials`.
```

Comment:
The underscores above in `_<action>_` and `_<option>_` look like holdovers from back when we were doing that for replaceables. Can be removed now, I think.